macros/REST_Axion_XMMAngleEff.C

```diff
@@ -0,0 +1,101 @@
+#include "TCanvas.h"
+#include "TLatex.h"
+#include "TLegend.h"
+#include "TRestAxionWolterOptics.h"
```

Comment:
It is not necessary to add the include of `TRestAxionWolterOptics` since it is not directly used. Also, even if it would be used probably is not needed in macros because using `restRootMacros` libraries and dictionaries will be loaded, and thus headers will be loaded too.